scripts/question_utils.py

```diff
@@ -5,63 +5,54 @@
 import pathlib
 from random import choice
 from typing import List
-
-p = pathlib.Path(__file__).parent.parent.joinpath('README.md')
+import os
+'''
+devops-exercises//README.md
+'''
+p = os.getcwd()+'/README.md'
```

Comment:
Don't you assume here that the script runs from the root of the repo? What happens if you don't run it from the root of the directory?

Reply:
you're right, i'll fix it,thanks.